src/main/java/core/basesyntax/dao/fileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.dao;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class fileWriterImpl implements fileWriter {
```

Comment:
The implemented interface `fileWriter` should be renamed to `FileWriter` to follow Java naming conventions.